### ISSUE TITLE
Minor chai upgrade truncates error messages

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#12348]
This ticket isn’t exactly high priority but we **may want to downgrade chai in the meantime** to prevent potential frustration with this.

See [this Slack thread](https://mozilla.slack.com/archives/CLV3KMZ8B/p1647535600317769).

A recent [chai upgrade](https://github.com/mozilla/fxa/pull/12091) from 4.3.4 to 4.3.6 strangely seems to truncate error messages. This makes certain test failures, like emails, impossible to see.

We don’t seem to have a chai config, but [the docs](https://www.chaijs.com/guide/styles/#configuration) say we can turn it off. Do we want to turn off truncation globally? If so, hopefully we can find a better way than something like [this](https://stackoverflow.com/questions/57230470/setting-chai-config-globally-once-for-all-tests).

!image-20220401-193549.png|width=1058,height=683!

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-4792)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/22436">image-20220401-193549.png</a>

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Let’s remove the truncation only for those long tests and also investigate if there is an option to have a local configuration file (like an environment variable) so everyone can set the option however they want.

**Comment by data-sync-user** [comment]
➤ Mill Soper commented:

Wil Clouser Dan Schomburg I was just talking about this with Lauren Zugai and she mentioned that this hasn’t been a problem anywhere except for in the email tests – we thought that it might be simplest (and most useful) to just remove truncation in the email tests, and add a note into documentation pointing to that change in case people want to follow that pattern elsewhere. I found that I could disable truncation with two lines of code in a test and setting a value on the package config. Does this sound like a reasonable approach?

**Comment by data-sync-user** [comment]
➤ Mill Soper commented:

TL;DR We could disable truncation for the whole package (or multiple packages) if desired but it seems like we don’t have a need for that across the board, so it might be easier to add documentation showing an example of how to disable truncation for blocks of tests.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Good idea.  r+
